operator/v1/0000_90_cluster_csi_driver_01_config.crd.yaml-patch

```diff
@@ -17,3 +17,4 @@
       - csi.sharedresource.openshift.io
       - diskplugin.csi.alibabacloud.com
       - vpc.block.csi.ibm.io
+      - csi.nutanix.com
```

Comment:
Nit, there should be a new line after this

Reply:
Ok i will add it
Thanks @JoelSpeed

Reply:
fixed in [18b340e](https://github.com/openshift/api/pull/1138/commits/18b340ed1d5f051eb97540ec93049dbbc6a26395)